peer.go

```diff
@@ -1093,6 +1093,18 @@ func (p *peer) handleGetHeadersMsg(msg *wire.MsgGetHeaders) {
 // filter.  The peer will be disconnected if a filter is not loaded when this
 // message is received.
 func (p *peer) handleFilterAddMsg(msg *wire.MsgFilterAdd) {
+	if p.server.services&wire.SFNodeBloom != wire.SFNodeBloom {
```

Comment:
This code section is identical with the exception of the command name in the debug message for all three functions.  It should probably be refactored out into a function.